cmd/controller/main.go

```diff
@@ -142,19 +143,14 @@ func main() {
 	logger.Info("Got SIGTERM signal")
 }
 
-func createReservationManager(client *k8s.KubeClient, log *logrus.Logger) (mgr ctrl.Manager, err error) {
+func createReservationManager(client *k8s.KubeClient, log *logrus.Logger, ch <-chan struct{}) (manager.Manager, error) {
 	// create scheme
-	scheme := runtime.NewScheme()
-	if err := clientgoscheme.AddToScheme(scheme); err != nil {
-		return nil, err
-	}
-
-	// register ACR CRD
-	if err = acrcrd.AddToSchemeACR(scheme); err != nil {
+	scheme, err := k8s.PrepareScheme()
```

Comment:
Why do we need to register for all CRDs?

Reply:
Fixed